neuralmonkey/decoders/ctc_decoder.py

```diff
@@ -120,7 +123,12 @@ def logits(self) -> tf.Tensor:
     def feed_dict(self, dataset: Dataset, train: bool = False) -> FeedDict:
         fd = ModelPart.feed_dict(self, dataset, train)
 
-        sentences = dataset.maybe_get_series(self.data_id)
+        sentences = cast(Iterable[List[str]],
+                         dataset.maybe_get_series(self.data_id))
+
+        if sentences is None and train:
+            raise ValueError("When training, you must feed "
+                             "reference sentences")
```

Comment:
@jindrahelcl co je to tady za OMG na řádce 147. Tam se do feeddictu přazuje nově udělanej node z grafu, kterej se vyrobí z numpy konstant.

Reply:
`tf.SparseTensorValue` neni symbolický  ale doopravdický.